config.json

```
@@ -28,6 +28,20 @@
         "strings"
       ]
     },
+    {
+      "slug": "high-scores",
+      "uuid": "574d6323-5ff5-4019-9ebe-0067daafba13",
+      "core": true,
```

```suggestion
      "core": false,
```

This is not a core exercise.